tools/integration/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "integration",
+  "version": "1.0.0",
+  "description": "Integration tests and related tools for ClearlyDefined",
+  "scripts": {
+    "test": "npm run mocha && npm run lint",
+    "mocha": "mocha --exit \"test/lib/**/*.js\"",
+    "lint": "npm run prettier:check && npm run eslint",
+    "lint:fix": "npm run prettier:write && npm run eslint:fix",
+    "eslint": "eslint .",
+    "eslint:fix": "eslint . --fix",
+    "prettier:check": "prettier . --check",
+    "prettier:write": "prettier . --write"
+  },
+  "author": "",
+  "license": "MIT",
+  "devDependencies": {
+    "chai": "^4.3.7",
```

Comment:
One thought I had today: I think with out current usage, we don't really need a third-party dependency for assertions and could just rely on the native `assert`. Nothing to necessarily address right now but maybe in the future.

Reply:
True.  That will reduce the maintenance effort in the future.